ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-dev.yaml

```diff
@@ -0,0 +1,49 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-odh-kueue
+  steps:
+    cluster_profile: aws-opendatahub
+    env:
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
+      HYPERSHIFT_AWS_REGION: us-west-2
+      HYPERSHIFT_NODE_COUNT: "3"
+    test:
+    - as: e2e-odh-kueue
+      cli: latest
+      commands: |
+        oc apply --server-side -k "github.com/ChughShilpa/kueue/config/rhoai?ref=dev"
```

Comment:
Can you install Kueue using ODH, referring to the ref from PR check or main branch?

Something similar as done in https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/codeflare-operator/opendatahub-io-codeflare-operator-main.yaml#L45-L53

That way you can test deployment scripts from PR checks (in future also image built from the source code, once image placeholder is supported in ODH).

Reply:
Added the change @sutaakar